doc-content/enterprise-only/springboot/bus-app-user-group_proc.adoc

```diff
@@ -0,0 +1,37 @@
+[id='bus-app-user-group_{context}']
+= Configuring business application user group providers
+
+Business automation capability supports the management of human-centric activities. To provide integration with user and group repositories, there are two KIE API entry points available:
+
+* `UserGroupCallback`: Responsible for verifying whether a user or group exists and for collecting groups for a specific user
+* `UserInfo`: Responsible for collecting additional information about users and groups, for example email addresses and preferred language
+
+Both of these components can be configured by providing alternative code, either code provided out of the box or custom developed code.
+
+For the `UserGroupCallback` component, it is recommended that you retain the default implementation because it is based on the security context of the application. It uses the back-end store, for example RH-SSO, for authentication and authorization and as source of information for collecting user and group information.
```

Comment:
@MarianMacik updated as suggested. However, can you tell me what 'It' is:
It will be automatically used as source of information for collecting user and group information.

Reply:
The default implementation of UserGroupCallback.